src/components/RightSidebar/SharedItems/sharedItemsConstants.ts

```diff
@@ -6,6 +6,7 @@ import { t } from '@nextcloud/l10n'
 import { SHARED_ITEM } from '../../../constants.ts'
 
 export const sharedItemsOrder = [SHARED_ITEM.TYPES.MEDIA,
+	SHARED_ITEM.TYPES.PINNED,
```

Comment:
Don't see a design notes on this, but IMO pinned messages should be listed above images
```suggestion
export const sharedItemsOrder = [
	SHARED_ITEM.TYPES.PINNED,
	SHARED_ITEM.TYPES.MEDIA,
```